docs/wiki/research/roadmap.md

```diff
@@ -74,74 +74,100 @@ Upgrades related to censorship resistance, decentralization and mitigating proto
+|------------------|-------------|----------------------------|----------------------------------------------|----------------------------------------------------|---------------------------------------------------------------------------------|
+| Increase MAX_EFFECTIVE_BALANCE | Staking Economics | Raising Validator Cap | Increase the max balance for Ethereum validators from 32 ETH to reduce overhead for large stakers | Consolidates validators, reduces network load, simplifies operations for large stakers | [in research](/wiki/research/eODS.md)[^28], confirmed for Pectra upgrade |
+| Improve Node Operator Usability | Staking Economics | Making Nodes Cheaper | Make nodes cheaper and easier to operate using verkle trees and SNARKs | Lower SSD requirements, faster sync times, easier node operation | Research/Proposal: [in eps node workshop](/docs/eps/nodes_workshop.md)[^29] |
+| Explore Total Stake Capping | Staking Economics | Capping Validator Set | Cap the total amount of stake to manage communication overhead between validators | Prevents excessive validator participation, maintains network efficiency | Research/Proposals: [in research](/wiki/research/eODS.md)[^30] |
+| Explore Solutions to Liquid Staking Centralization | Staking Economics | Combat LST Centralization | Solutions to reduce centralization in the Liquid Staking Token (LST) market | Prevents large LST providers from gaining too much control over the network | Research/Proposals: [^31], [^32], [^33], [^34],[^35] |
+
+
+                                              
+### The Verge
 Upgrades related to verifying blocks more easily
 
 | Upgrade | Track | Topic | Description | Expected effect | State of the art |
 | :------ | :---: | :---: | :---------: | :-------------: | :--------------- |
 |         |       |       |             |                 |                  |
 
-### the Purge
+### The Purge
 Upgrades related to reducing the computational costs of running nodes and simplifying the protocol
 
-### the Splurge
+### The Splurge
 Other upgrades that don't fit well into the previous categories.
 
 ## Resources
 
-[^1]: EIP-2982: Serenity Phase 0 https://eips.ethereum.org/EIPS/eip-2982, [[archived]](https://web.archive.org/web/20230928204358/https://eips.ethereum.org/EIPS/eip-2982)
+[^1]: [EIP-2982: Serenity Phase 0](https://eips.ethereum.org/EIPS/eip-2982), [[archived]](https://web.archive.org/web/20230928204358/https://eips.ethereum.org/EIPS/eip-2982)
+
+[^2]: [EIP-4895: Beacon chain push withdrawals](https://eips.ethereum.org/EIPS/eip-4895), [[archived]](https://web.archive.org/web/20240415201815/https://eips.ethereum.org/EIPS/eip-4895)
+
+[^4]: [VB's SSF notes](https://notes.ethereum.org/@vbuterin/single_slot_finality), [[archived]](https://web.archive.org/web/20240330010706/https://notes.ethereum.org/@vbuterin/single_slot_finality)
```

Comment:
Hi @gconnect thanks for improving this.

I noticed that the foot note is not enumerated correctly. no. 3 is missing.